src/vaadin-custom-field-mixin.html

```diff
@@ -150,19 +150,25 @@
       this.__settingValue = false;
     }
 
+    /* Like querySelectorAll('*') but also gets all elements through any nested slots recursively */
+    __queryAllAssignedElements(elem) {
+      const result = [];
+      let elements;
+      if (elem.tagName === 'SLOT') {
+        elements = elem.assignedNodes({flatten: true}).filter(node => node.nodeType === Node.ELEMENT_NODE);
+      } else {
+        result.push(elem);
+        elements = Array.from(elem.children);
+      }
+      elements.forEach(elem => {
+        result.push(...this.__queryAllAssignedElements(elem));
```

Comment:
Non-blocking nit: is it possible to make one-liner? :)

Reply:
You mean like this just removing few new lines?
```js
elements.forEach(elem => { result.push(...this.__queryAllAssignedElements(elem)); });
```
Or something else?

Reply:
Something like:
`elements.forEach(elem => result.push(...this.__queryAllAssignedElements(elem)));`

Reply:
Updated. Will merge after tests pass.